app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminCreateCustomerAndSubscribeActionGroup.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It is better to split this Ag into two:
1) create customer
2) setup subscription